misc/ls-extensions/modules/sa-modeling-service/src/main/java/io/ballerina/componentmodel/ComponentModelingConstants.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.componentmodel;
+
+import io.ballerina.compiler.syntax.tree.SyntaxKind;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Constants use for Solution Architecture model generation.
+ */
+public class ComponentModelingConstants {
+
+    /**
+     * Enum to select the type of the parameter.
+     */
+    public enum ParameterIn {
+        BODY("body"),
+        QUERY("query"),
+        HEADER("header"),
+        PATH("path");
+
+        private final String parameterIn;
+
+        ParameterIn(String parameterIn) {
+
+            this.parameterIn = parameterIn;
+        }
+
+        public String getValue() {
+
+            return this.parameterIn;
+        }
```

Comment:
Shall we remove additional blank line after the method signatures from the method definitions? (specially in getters/setter)

Reply:
Only removed in getters and setters.